x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -13,6 +13,10 @@ import { Logger } from '@kbn/core/server';
 import { ConcreteTaskInstance, throwUnrecoverableError } from '@kbn/task-manager-plugin/server';
 import { nanosToMillis } from '@kbn/event-log-plugin/server';
 import { DEFAULT_NAMESPACE_STRING } from '@kbn/core-saved-objects-utils-server';
+import {
+  createTaskRunError,
+  TaskErrorSource,
+} from '@kbn/task-manager-plugin/server/task_running/errors';
```

Comment:
nit: should we export this from the `task_manager/server/index.ts` file so we can just import from `@kbn/task-manager-plugin/server?`